Makefile

```diff
@@ -150,20 +150,20 @@ tests:
 	(cd test/stdtypes && make test)
 
 vet:
-	go get golang.org/x/tools/go/analysis/passes/shadow/cmd/shadow
+	go install golang.org/x/tools/go/analysis/passes/shadow/cmd/shadow@latest
 	go vet ./...
 	go vet -vettool=$(shell which shadow) ./...
 
 errcheck:
-	go get github.com/kisielk/errcheck
+	go install github.com/kisielk/errcheck@latest
 	errcheck ./test/...
 
 drone:
 	sudo apt-get install protobuf-compiler
 	(cd $(GOPATH)/src/github.com/gogo/protobuf && make buildserverall)
 
 testall:
-	go get -u github.com/golang/protobuf/proto
+	go install github.com/golang/protobuf/proto@latest
```

Comment:
let's pin this to a specific version.

Reply:
Reverted this, it didn't work since it's not a binary. It's not being called in CI.